editor.planx.uk/src/@planx/components/shared/Schema/InputFields/MapFieldInput.tsx

```diff
@@ -11,6 +12,19 @@ import { getFieldProps, Props } from ".";
 import { FieldInputDescription } from "./shared";
 
 export const MapFieldInput: React.FC<Props<MapField>> = (props) => {
+  // Ensure there's a FindProperty component preceding this field (eg address data in state to position map view)
+  const { longitude, latitude } = useStore(
+    (state) =>
+      (state.computePassport()?.data?.["_address"] as SiteAddress) || {},
+  );
+
+  if (!longitude || !latitude) {
+    throw Error(
+      'Edit this flow so that this component is positioned after "FindProperty"; an address is required for schemas that include a "map" field.',
+      { cause: "Invalid graph" },
+    );
```

Comment:
nit: I'm not sure I'm sold on throwing an error here rather than rendering a component as before - do we want this noise coming through Airbrake? 

Understand we can't easily render the whole component any longer from this shared position, but just wondering if there's a mechanism to exclude from Airbrake here etc?